tool/site/client/src/pages/sendPostSignPage.tsx

```diff
@@ -75,13 +79,25 @@ const SendPostSignPage: React.FC<SendPostSignPageProps> = ({
   )
 }
 
-async function sendPrescription(baseUrl: string, token: string, state?: string): Promise<SendResult | SendBulkResult> {
+async function sendPrescription(
+  baseUrl: string,
+  token: string,
+  state?: string
+): Promise<SendResult | SendBulkResult | Redirect> {
   const request = {signatureToken: token, state}
-  const response = await axiosInstance.post<SendResult | SendBulkResult>(`${baseUrl}prescribe/send`, request)
+  const response = await axiosInstance.post<SendResult | SendBulkResult | Redirect>(`${baseUrl}prescribe/send`, request)
+  if (isRedirect(response.data)) {
+    redirect(response.data.redirectUri)
+    return response.data
+  }
   return getResponseDataIfValid(response, isSendResultOrSendBulkResult)
 }
 
-function isSendResultOrSendBulkResult(data: unknown): data is SendResult | SendBulkResult {
+function isRedirect(data: unknown): data is Redirect {
```

Comment:
might be useful to have this typeguard in a common folder in the future?

Reply:
if this works when deployed i'll raise another pr to move it